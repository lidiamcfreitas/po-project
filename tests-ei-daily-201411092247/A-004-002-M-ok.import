USER|vader|Darth Vader|/home/vader
USER|yoda|Master Yoda|/home/yoda
DIRECTORY|/home/root/luke|root|private
DIRECTORY|/home/root/vader|root|private
DIRECTORY|/home/root/yoda|root|private
