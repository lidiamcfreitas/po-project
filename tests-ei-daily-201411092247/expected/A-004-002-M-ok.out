Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: Menu de Shell
1 - Listar
2 - Listar entrada
3 - Remover entrada
4 - Mudar o directório de trabalho
5 - Criar ficheiro
6 - Criar directório
7 - Mostrar o directório de trabalho
8 - Adicionar linha a ficheiro
9 - Ver conteúdo de ficheiro
10 - Mudar permissões de entrada
11 - Mudar dono de entrada
0 - Sair
Escolha uma opção: d - root 40 .
d - root 40 ..
d - root 16 luke
d - root 16 vader
d - root 16 yoda
Menu de Shell
1 - Listar
2 - Listar entrada
3 - Remover entrada
4 - Mudar o directório de trabalho
5 - Criar ficheiro
6 - Criar directório
7 - Mostrar o directório de trabalho
8 - Adicionar linha a ficheiro
9 - Ver conteúdo de ficheiro
10 - Mudar permissões de entrada
11 - Mudar dono de entrada
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: 